### PR TITLE
More renaming

### PR DESCRIPTION
**Pull request by AndiMiko**
**Type of change:**
- [ ] Feature work
- [ ] Bug fix
- [ ] Documentation
- [X] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

**Risk**:
- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [X] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

File: sdk/src/main/java/com/microsoft/did/sdk/utilities/DefaultLogConsumer.kt [hunk 1]
```diff
@@ -2,7 +2,7 @@ package com.microsoft.did.sdk.utilities
 
 import android.util.Log
 
-class DefaultLogConsumerBridge: SdkLog.ConsumerBridge {
+class DefaultLogConsumer: SdkLog.Consumer {
```
Comment (nithyaganeshng) [inline sdk/src/main/java/com/microsoft/did/sdk/utilities/DefaultLogConsumer.kt:5]: Just curious, is there a reason we are moving away from bridge in naming?

File: sdk/src/main/java/com/microsoft/did/sdk/di/SdkModule.kt [hunk 2]
```diff
@@ -110,7 +110,7 @@ internal class SdkModule {
     @Provides
     @Singleton
     fun sdkDatabase(context: Context): SdkDatabase {
-        return Room.databaseBuilder(context, SdkDatabase::class.java, "PortableIdentity-db")
+        return Room.databaseBuilder(context, SdkDatabase::class.java, "VerifiableCredential-db")
             .fallbackToDestructiveMigration() // TODO: we don't want this here as soon as we go into production
```
Comment (nithyaganeshng) [inline sdk/src/main/java/com/microsoft/did/sdk/di/SdkModule.kt:114]: Just noticed the comment, is it something we have to remove in this sprint?

File: sdk/src/main/java/com/microsoft/did/sdk/utilities/DefaultLogConsumer.kt (hunk 1 above)
Comment (AndiMiko) [reply sdk/src/main/java/com/microsoft/did/sdk/utilities/DefaultLogConsumer.kt:5]: I was just renaming bunch of stuff and noticed it's unnecessarily verbose and `(Log)Consumer` is already perfectly descriptive :P

File: sdk/src/main/java/com/microsoft/did/sdk/di/SdkModule.kt (hunk 2 above)
Comment (AndiMiko) [reply sdk/src/main/java/com/microsoft/did/sdk/di/SdkModule.kt:114]: That's a good call. I think as of yet it is debatable whether we are "in production" in this sense. Essentially we are still somewhat ok to wipe peoples databases (that's what it does) when something changes. At some point we are totally not ok with that anymore obviously 😂

**Pull request merged**